### ISSUE TITLE
[Bug]: Docker Image not working

### ISSUE THREAD

**Issue by om-surushe** [zitadel/zitadel#6814]
### Preflight Checklist

- [X] I could not find a solution in the documentation, the existing issues or discussions
- [X] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Environment

Self-hosted

### Version

v2.39.1

### Describe the problem caused by this bug

Whenever I go to http://localhost:8080/ui/console, I get "ID=QUERY-1kIjX Message=Instance not found".
I am not able to login to Zitadel console.

### To reproduce

1. Download the docker compose example configuration.
`wget https://raw.githubusercontent.com/zitadel/zitadel/main/docs/docs/self-hosting/deploy/docker-compose.yaml`

2. Run the database and application containers.
`docker compose up --detach`

3. Check if Zitadel version is v2.39.1 and CRDB image is v22.2.2
 
4. Visit 
[http://localhost:8080/ui/console](http://localhost:8080/ui/console)

### Screenshots

![Screenshot from 2023-10-26 10-22-18](https://github.com/zitadel/zitadel/assets/83531291/8a935570-bdfc-4ffa-b8b1-14b9de52627c)

### Expected behavior

It was working fine on my windows 11, but not properly working on Ubuntu 20.04

### Operating System

Ubuntu 20.04
Docker version 24.0.6, build ed223bc

### Additional Context

This was the line in the log

zitadel-zitadel-1  | time="2023-10-26T04:43:53Z" level=info msg="query failed" caller="/home/runner/work/zitadel/zitadel/internal/eventstore/repository/sql/query.go:113" error="ERROR: only users with the admin role are allowed to read crdb_internal.cluster_transactions (SQLSTATE 42501)"

**Comment by hifabienne** [comment]
Hei @om-surushe 
Did you check out the following page: https://zitadel.com/docs/self-hosting/deploy/troubleshooting#instance-not-found

**Comment by om-surushe** [comment]
> Hei @om-surushe Did you check out the following page: https://zitadel.com/docs/self-hosting/deploy/troubleshooting#instance-not-found

Yes, and I am trying to setup locally for local development.

**Comment by hifabienne** [comment]
@eliobischof Can you help here?

**Comment by eliobischof** [comment]
@om-surushe can you reproduce it with the latest cockroach version?

**Comment by om-surushe** [comment]
> @om-surushe can you reproduce it with the latest cockroach version?

The issue is with cockroach:v22.2.2

**Comment by eliobischof** [comment]
Can you upgrade crdb and try again plz?

**Comment by FedericoAntoniazzi** [comment]
Hello there, I'm joining this conversation because I've been trying out ZITADEL in a lab environment for few months now and in the last period I faced the same issue.

I'm not able to log in into the services which I configured to use ZITADEL as oauth provider for the same reason reported by @om-surushe.

I used the same configuration reported by the documentation and got the same error. I even tried to upgrade gradually to latest version of both ZITADEL and Cockroachdb but nothing changes.

In addition to the info the author of the issue has already share, I want to say that I haven't touched the setup for weeks and suddenly faced the issue.

I'm available for helping troubleshooting the error and to provide any useful information from my broken environment :D
